charts/linkerd-control-plane/templates/identity.yaml

```
@@ -215,6 +215,8 @@ spec:
       {{- $_ := set $tree.Values.proxy "capabilities" (dict "drop" (list "ALL")) }}
       {{- $_ := set $tree.Values.proxy "outboundDiscoveryCacheUnusedTimeout" "5s" }}
       {{- $_ := set $tree.Values.proxy "inboundDiscoveryCacheUnusedTimeout" "90s" }}
+      {{- $_ := set $tree.Values.proxy "inboundHTTP1ConnectionPoolIdleTimeout" "3s" }}
+      {{- $_ := set $tree.Values.proxy "outBoundHTTP1ConnectionPoolIdleTimeout" "3s" }}
```

This is a bit nitpicky but we generally don't capitalise the B in outbound.

```suggestion
      {{- $_ := set $tree.Values.proxy "outboundHTTP1ConnectionPoolIdleTimeout" "3s" }}
```

---

Good catch, updated

---

`cli/cmd/doc.go` has been also updated.